### PR TITLE
Add a new tool to move info such as signature and product/component f…

### LINKED ISSUE #184: Move crash signatures in duplicate bug
For example, if bug 1 has signature "foo::bar()" and it's marked duplicate of bug 2 then check that this signature is in bug 2 too (probably auto-fix here).

### LINKED ISSUE #225: Automatic cleanup of DUPLICATE bugs
We can do a few things:
- [ ] Copy the signatures from the bug resolved as DUPLICATE to the bug it has been duplicated to (#184)
- [ ] Update the component of the DUPLICATE bug making it the same as the bug it has been duplicated to

### PR DESCRIPTION
**Pull request by calixteman**
…or bugs and their dups

Fixes #184 and fixes #225.

### CODE REVIEW

File: auto_nag/scripts/duplicate.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+from auto_nag import utils
+
+
+class Duplicate(BzCleaner):
+    def __init__(self):
+        super(Duplicate, self).__init__()
+        self.autofix_data = {}
+
+    def description(self):
+        return 'Get the closed bugs with stalled keyword'
```
Comment (sylvestre) [inline auto_nag/scripts/duplicate.py]: I think this is wrong :)

File: auto_nag/scripts/duplicate.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+    def name(self):
+        return 'duplicate'
+
+    def template(self):
+        return 'duplicate.html'
+
+    def subject(self):
+        return 'Closed bugs with stalled keyword'
```
Comment (sylvestre) [inline auto_nag/scripts/duplicate.py]: ditto

File: runauto_nag_hourly.sh [hunk 3]
```diff
@@ -64,5 +64,9 @@ python -m auto_nag.scripts.mismatch-priority-tracking-release
 python -m auto_nag.scripts.mismatch-priority-tracking-beta
 python -m auto_nag.scripts.mismatch-priority-tracking-nightly
 
+# Move info (signatures, product/component) from/to bugs & their dups
+# Pretty common
+python2 -m auto_nag.scripts.duplicate
```
Comment (sylvestre) [inline runauto_nag_hourly.sh]: duplicate crash ?

Comment (calixteman) [reply runauto_nag_hourly.sh]: what do you mean ?

Comment (sylvestre) [reply runauto_nag_hourly.sh]: duplicate is too generic.
so, I would prefer that we make it specific to crash
renaming for duplicate_crash for example

Comment (calixteman) [reply runauto_nag_hourly.sh]: yeah you're right, but I'm moving product/info too (from original to dup)
So duplicate_bug maybe ?

Comment (marco-c) [reply runauto_nag_hourly.sh]: Maybe `copy_duplicate_info`?

Comment (calixteman) [reply runauto_nag_hourly.sh]: wfm, @sylvestre ?

Comment (sylvestre) [reply runauto_nag_hourly.sh]: Works for me

**Pull request merged**